BrandonBeltran/hackerrank/03/RansomNote.java

```diff
@@ -0,0 +1,38 @@
+package bootcamp;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+import java.util.stream.Stream;
+import static java.util.stream.Collectors.toList;
+
+public class RansomNote {
+
+    public static final String WHITESPACE_AND_EOL = "\\s+$";
+
+    public static void main(String[] args) throws IOException {
+
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
+        String[] totalWords = bufferedReader.readLine().replaceAll(WHITESPACE_AND_EOL, "").split(" ");
+
+        int magazineTotalWords = Integer.parseInt(totalWords[0]);
+        int noteTotalWords = Integer.parseInt(totalWords[1]);
+
+        List<String> magazine = Stream.of(bufferedReader.readLine().replaceAll(WHITESPACE_AND_EOL, "").split(" "))
```

Comment:
Best practice in Collections or Map Variable Name, is to use singular name and them interface name: numberList, change it in all variables